src/modules/utils/services/MediaStream.js

```diff
@@ -23,15 +23,34 @@
 
         }
 
-        function getFirstVideoDevice(list) {
-            let item = null;
-            list.some((device) => {
-                if (device.kind === 'videoinput') {
-                    item = device;
+        function getFacingBackCamera(devices) {
+            let facingBackCamera = devices.find((device) => device.label.includes('back'));
+
+            if (!facingBackCamera) {
+                const videoInputDevices = devices.filter((device) => device.kind === 'videoinput');
```

Comment:
Add polyfill or replace to "tsUtils.find(devices, (device) => device.kind === 'videoinput')"